scripts/remove-notify-app-ready.mjs

```
@@ -0,0 +1,238 @@
+ * @param {boolean} dryRun
+ */
+function deleteEmptyNuxtPlugin(targetDir, dryRun) {
+  const candidates = [
+    join(targetDir, 'plugins', 'capacitorUpdater.client.ts'),
+    join(targetDir, 'plugins', 'capacitorUpdater.client.js'),
+  ]
+  const removed = []
+  for (const file of candidates) {
+    try {
+      const contents = readFileSync(file, 'utf8')
+      const trimmed = contents.replace(/\s+/g, '')
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)